common/src/algos/et.rs

```diff
@@ -23,14 +37,31 @@ fn pre_process(lt: &mut HashMap<(u64, u64), f32>) {
 }
 
 /// Normalizes the `lt` matrix by dividing each element by the sum of its row.
-fn normalise_lt(lt: &mut HashMap<(u64, u64), f32>) -> Result<(), algos::Error> {
+fn normalise_lt(
+    all_peers: &HashSet<u64>, lt: &mut HashMap<(u64, u64), f32>, seed: &HashMap<u64, f32>,
+) -> Result<(), algos::Error> {
+    // Calculate seed sum for later use
+    let seed_sum: f32 = seed.iter().map(|(_, v)| v).sum();
+
     // Calculate the sum of each row in the local trust matrix.
     let mut sum_map: HashMap<u64, f32> = HashMap::new();
     for ((from, _), value) in lt.iter() {
         let val = sum_map.get(from).unwrap_or(&0.0);
         sum_map.insert(*from, val + value);
     }
 
+    for from in all_peers.iter() {
+        let sum = sum_map.get(from).unwrap_or(&0.0);
+        // If peer does not have outbound trust,
+        // his trust will be distributed to seed peers based on their seed/pre-trust
+        if *sum == 0.0 {
+            for (to, value) in seed {
+                lt.insert((*from, *to), *value);
+            }
```

Comment:
I have a question.
If `from` == `to`, it means giving trust value to itself.
Then, we should call `pre_process` after `normalize_lt`. 
Is it right?